store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java

```diff
@@ -872,6 +872,10 @@ public GetMessageResult getMessage(final String group, final String topic, final
                                 continue;
                             }
 
+                            if (messageStoreConfig.isColdDataFlowControlEnable() && !selectResult.isInCache()) {
```

Comment:
System Consumer Groups(such as TOOLS_CONSUMER_GROUP) need to be considered, to prevent them from being limited.

Reply:
yes, good idea~